numba/tests/test_parfors.py

```diff
@@ -3499,6 +3499,11 @@ def pyfunc():
 
         with warnings.catch_warnings(record=True) as raised_warnings:
             warnings.simplefilter('always')
```

Comment:
My "guess" would be, that this reset's all warnings, so they are always shown (even if we asked for something else via an environment variable or on the command line). So my question is, can this line not just be deleted?

Reply:
Otherwise, this seems sane. It means parfors won't fall over because too many tbb interface version warnings are raised.

Reply:
IIRC `'always'` means keep on printing them even if you've seen them before, this PR is just adding a new rule to ignore a particular class of warning.

Reply:
I can confirm that the following `filterwarnings` is appending a filter that will suppress the TBB warning. I did the same in the typeguard PR.

Reply:
ref: https://github.com/numba/numba/pull/6751/commits/a4ff69b49e5675b94ed883068775b59427653548